src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2586,6 +2586,104 @@ public function get_test_persistent_object_cache() {
 		return $result;
 	}
 
+	/**
+	 * Calculate total amount of autoloaded data.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @return int autoloaded data in bytes.
+	 */
+	public function get_autoloaded_options_size(): int {
+		$alloptions = wp_load_alloptions();
+
+		$total_length = 0;
+
+		foreach ( $alloptions as $option_name => $option_value ) {
+			$total_length += strlen( $option_value );
+		}
+
+		return $total_length;
+	}
+
+	/**
+	 * Callback for autoloaded_options test.
```

Comment:
```suggestion
	 * Tests the number of autoloaded options.
```
- Methods should begin with a third-person singular verb.
- This suggestion follows the example of other callbacks in this file.